packages/react-vis/tests/components/area-series-tests.js

```diff
@@ -32,12 +32,12 @@ test('AreaSeries: basic rendering', t => {
     'should find the right number of series'
   );
   t.equal(
-    $.find('.rv-xy-plot__series path').length,
+    $.find('path.rv-xy-plot__series').length,
```

Comment:
This change seems odd. Did we change the render logic?

Reply:
I didn't touch the render logic, but my guess is that it has to do with newer versions of enzyme (didn't look too deeply) and how it handles the css selectors.

The original `$.find('.rv-xy-plot__series path').length` was returning 0 elements.

Without the additional `path` selector we get this:
```
// $.find('.rv-xy-plot__series').debug()
<path d="M0,250L125,0L250,166.66666666666669L250,183.33333333333334L125,150L0,233.33333333333334Z" className="rv-xy-plot__series rv-xy-plot__series--line area-chart-example" transform="translate(40,10)" onMouseOver={[Function]} onMouseOut={[Function]} onClick={[Function]} onContextMenu={[Function]} style={{...}} />
```

The root node is rendered as
```
<XYPlot width={300} height={300} className="">
  <div style={{...}} className="rv-xy-plot">
    <svg className="rv-xy-plot__inner" width={300} height={300} style={[undefined]} onClick={[Function]} onDoubleClick={[Function]} onMouseDown={[Function]} onMouseUp={[Function]} onMouseMove={[Function]} onMouseLeave={[Function]} onMouseEnter={[Function]} onTouchStart={[Function]} onTouchMove={[Function]} onTouchEnd={[Function]} onTouchCancel={[Function]} onWheel={[undefined]}>
      <AreaSeries className="area-chart-example" color="#12939a" data={{...}} stack={false} style={{...}} getNull={[Function: getNull]} marginLeft={40} marginTop={10} marginRight={10} marginBottom={40} innerHeight={250} innerWidth={250} animation={[undefined]} sameTypeTotal={1} sameTypeIndex={0} clusters={{...}} seriesIndex={0} _colorValue="#12939a" _opacityValue={1} xRange={{...}} yRange={{...}} colorRange={{...}} fillRange={{...}} strokeRange={{...}} opacityType="literal" sizeRange={{...}} getX={[Function]} getX0={[Function]} xDomain={{...}} getY={[Function]} getY0={[Function]} yDomain={{...}} getRadius={[Function]} getRadius0={[Function]} radiusDomain={{...}} getAngle={[Function]} getAngle0={[Function]} angleDomain={{...}} getColor={[Function]} getColor0={[Function]} colorDomain={{...}} getFill={[Function]} getFill0={[Function]} fillDomain={{...}} getStroke={[Function]} getStroke0={[Function]} strokeDomain={{...}} getOpacity={[Function]} getOpacity0={[Function]} opacityDomain={{...}} getSize={[Function]} getSize0={[Function]} sizeDomain={{...}} _allData={{...}} _adjustBy={{...}} _adjustWhat={{...}} _stackBy={[undefined]}>
        <path d="M0,250L125,0L250,166.66666666666669L250,183.33333333333334L125,150L0,233.33333333333334Z" className="rv-xy-plot__series rv-xy-plot__series--line area-chart-example" transform="translate(40,10)" onMouseOver={[Function]} onMouseOut={[Function]} onClick={[Function]} onContextMenu={[Function]} style={{...}} />
      </AreaSeries>
    </svg>
  </div>
</XYPlot>
```

Which makes sense given that the props that are passed to the area are
```
const AREA_PROPS = {
  className: 'area-chart-example',
  color: '#12939a',
  data: [
    {x: 1, y: 5, y0: 6},
    {x: 2, y: 20, y0: 11},
    {x: 3, y: 10, y0: 9}
  ]
};
```

The original version of the test was expecting that the `rv-xy-plot__series` className was provided directly to the AreaSeries, which it wasn't

Reply:
I ran the tests on master and they all passed. (details below)

It looks like the rendering stayed the same, but the evaluation of the selector changed.
The new way (in the `yarn-workspace` branch) makes more sense.

The selector `.rv-xy-plot__series path` means find a `path` that is a child of an element with the `rv-xy-plot__series` class.
The correct selector for "find a `path` that has the class `rv-xy-plot__series`" is `path.rv-xy-plot__series`

These are the results on master.
```
// console.log($.find('.rv-xy-plot__series path').debug())
<path d="M0,250L125,0L250,166.66666666666669L250,183.33333333333331L125,150L0,233.33333333333334Z" className="rv-xy-plot__series rv-xy-plot__series--line area-chart-example" transform="translate(40,10)" onMouseOver={[Function]} onMouseOut={[Function]} onClick={[Function]} onContextMenu={[Function]} style={{...}} />
```

```
// console.log($.find('.rv-xy-plot__series').debug())
<path d="M0,250L125,0L250,166.66666666666669L250,183.33333333333331L125,150L0,233.33333333333334Z" className="rv-xy-plot__series rv-xy-plot__series--line area-chart-example" transform="translate(40,10)" onMouseOver={[Function]} onMouseOut={[Function]} onClick={[Function]} onContextMenu={[Function]} style={{...}} />
```

```
// console.log($.debug())
<XYPlot width={300} height={300} className="">
  <div style={{...}} className="rv-xy-plot">
    <svg className="rv-xy-plot__inner" width={300} height={300} style={[undefined]} onClick={[Function]} onDoubleClick={[Function]} onMouseDown={[Function]} onMouseUp={[Function]} onMouseMove={[Function]} onMouseLeave={[Function]} onMouseEnter={[Function]} onTouchStart={[Function]} onTouchMove={[Function]} onTouchEnd={[Function]} onTouchCancel={[Function]} onWheel={[undefined]}>
      <AreaSeries className="area-chart-example" color="#12939a" data={{...}} stack={false} style={{...}} getNull={[Function]} marginLeft={40} marginTop={10} marginRight={10} marginBottom={40} innerHeight={250} innerWidth={250} animation={[undefined]} sameTypeTotal={1} sameTypeIndex={0} clusters={{...}} seriesIndex={0} _colorValue="#12939a" _opacityValue={1} xRange={{...}} yRange={{...}} colorRange={{...}} fillRange={{...}} strokeRange={{...}} opacityType="literal" sizeRange={{...}} getX={[Function]} getX0={[Function]} xDomain={{...}} getY={[Function]} getY0={[Function]} yDomain={{...}} getRadius={[Function]} getRadius0={[Function]} radiusDomain={{...}} getAngle={[Function]} getAngle0={[Function]} angleDomain={{...}} getColor={[Function]} getColor0={[Function]} colorDomain={{...}} getFill={[Function]} getFill0={[Function]} fillDomain={{...}} getStroke={[Function]} getStroke0={[Function]} strokeDomain={{...}} getOpacity={[Function]} getOpacity0={[Function]} opacityDomain={{...}} getSize={[Function]} getSize0={[Function]} sizeDomain={{...}} _allData={{...}} _adjustBy={{...}} _adjustWhat={{...}} _stackBy={[undefined]}>
        <path d="M0,250L125,0L250,166.66666666666669L250,183.33333333333331L125,150L0,233.33333333333334Z" className="rv-xy-plot__series rv-xy-plot__series--line area-chart-example" transform="translate(40,10)" onMouseOver={[Function]} onMouseOut={[Function]} onClick={[Function]} onContextMenu={[Function]} style={{...}} />
      </AreaSeries>
    </svg>
  </div>
</XYPlot>
```